src/components/structures/UserMenu.tsx

```diff
@@ -140,6 +146,29 @@ export default class UserMenu extends React.Component<IProps, IState> {
         );
     }
 
+    private checkQrLoginSupport = async (): Promise<void> => {
+        if (!this.context.client) return;
+
+        const { issuer } = await this.context.client.getAuthIssuer().catch(() => ({ issuer: undefined }));
```

Comment:
@t3chguy perhaps we should also be checking for the native OIDC labs flag?

This would mean that the menu item would only ever be visible if the labs flag is on. Currently it shows (as disabled) for everyone.

Reply:
Hm indeed that was the intent, wonder if this broke is a merge conflict resolution

Reply:
Ah so the check is done deeper in, `shouldShowQr` is the thing responsible for the check